src/rdkafka_int.h

```diff
@@ -234,6 +234,14 @@ rd_kafka_txn_state2str(rd_kafka_txn_state_t state) {
         return names[state];
 }
 
+typedef enum {
+        RD_KAFKA_TELEMETRY_AWAIT_BROKER,
```

Comment:
Don't we need to add state for GET_THROTTLED, PUSH_THROTTLED and also for the error cases?